src/main/java/org/servantscode/commons/db/ConfigDB.java

```diff
@@ -16,41 +16,34 @@
     private static final Logger LOG = LogManager.getLogger(ConfigDB.class);
 
     public String getConfiguration(String config) {
-        QueryBuilder queryBuilder = new QueryBuilder();
-        queryBuilder.select("value");
-        queryBuilder.from("configuration");
-        queryBuilder.where("config=?");
+        QueryBuilder queryBuilder = select("value").from("configuration").where("config=?", config);
         try (Connection conn = getConnection();
-             PreparedStatement stmt = queryBuilder.prepareStatement(conn)) {
+             PreparedStatement stmt = queryBuilder.prepareStatement(conn);
+             ResultSet rs = stmt.executeQuery()) {
+
+            if (rs.next())
+                return rs.getString(1);
 
-            stmt.setString(1, config);
-            try (ResultSet rs = stmt.executeQuery()) {
-                if (rs.next())
-                    return rs.getString(1);
-            }
         } catch (SQLException e) {
             throw new RuntimeException("Could not retrieve configuration property: " + config, e);
         }
         return null;
     }
 
     public Map<String, String> getConfigurations(String configPrefix) {
-        QueryBuilder queryBuilder = new QueryBuilder();
-        queryBuilder.select("*");
-        queryBuilder.from("configuration");
-        queryBuilder.where("config LIKE ?");
+        QueryBuilder queryBuilder = select("*").from("configuration").where("config LIKE ?", configPrefix);

```

Comment:
'%' is a wildcard in SQL. Allows for the matching of a leading portion of the field's value. This needs to be included in the .where().